spec/factories/book_factory.rb

```diff
@@ -0,0 +1,12 @@
+require 'faker'
+
+FactoryBot.define do
+  factory :book, class: Book do
+    genre { Faker::Book.genre }
+    author { Faker::Book.author }
+    image { Faker::Simpsons.quote }
+    title { Faker::Book.title }
+    editor { Faker::Book.publisher }
+    year { Faker::Date.between(2.days.ago, Time.zone.today).to_s[0, 4] }
```

Comment:
I find something weird about this,  because, in the validation you said that should validate that is an integer value, but here you are setting the year as a string